app/templates/content/index.hbs

```diff
@@ -44,6 +44,9 @@
             </div>
         </div> {{!END CONTAINER DIV}}
     </div> {{!END HEADER ROW}}
+    {{#if isAdmin}}
```

Comment:
This might be post-MVP, but as a moderator I would expect at least a link to the moderation view, if not a status bar like this.